src/build/ci-code-style.sh

```diff
@@ -1,8 +1,6 @@
 #!/bin/sh
 set -xe
 
-OPTS="-DskipTests=true -Dmaven.javadoc.skip=true -B"
```

Comment:
I think it was intentional that we skip building docs here, since that's checked in the `ci-doc` step

Reply:
that is the default in the basepom `<basepom.javadoc.skip>true</basepom.javadoc.skip>`